src/cm/config/config.hpp

```diff
@@ -44,10 +44,10 @@ struct UMController {
  * Monitoring configuration.
  */
 struct Monitoring {
-    monitoring::Config mMonitorConfig;
-    int                mMaxOfflineMessages;
-    Duration           mSendPeriod;
-    int                mMaxMessageSize;
+    aos::monitoring::Config mMonitorConfig;
+    int                     mMaxOfflineMessages;
```

Comment:
Looks like `mMaxOfflineMessages`, `mMaxMessageSize` are not needed any more.
`mSendPeriod` is part of aos::cm::monitoring::Config. I expect here something like this:

```c++
struct Monitoring : public aos::monitoring::Config, public aos::cm::monitoring::Config {
};
```